clang/test/Driver/sycl-offload.cpp

```diff
@@ -0,0 +1,15 @@
+/// Check "-aux-target-cpu" is passed when compiling for SYCL offload device :
+//  RUN:  %clang -### -fsycl -c %s 2>&1 | FileCheck -check-prefix=CHECK-TARG %s
+//  CHECK-TARG: "-aux-target-cpu" "x86-64"
+
+/// Check "-aux-target-cpu" is passed when compiling for SYCL offload device with enabling macros:
+//  RUN:  %clang -fsycl -mavx -c %s -### -o %t.o 2>&1 | FileCheck -check-prefix=TARG-AVX %s
+//  TARG-AVX: "-aux-target-cpu" "x86-64" "-aux-target-feature" "+avx"
+
+/// Check "target-cpu" is passed when compiling for SYCL offload host :
```

Comment:
This comment is misleading. The command below doesn't use SYCL mode, so it can't check "compiling for SYCL offload host".
I suggest merging last two checks into first two checks.

Reply:
Thanks for catching this. I have merged the checks now.